src/Umbraco.Core/Strings/Utf8ToAsciiConverter.cs

```diff
@@ -3560,29 +3560,33 @@ private static void ToAscii(ReadOnlySpan<char> input, int ipos, Span<char> outpu
                     output[opos++] = 'a';
                     break;
 
+
                 // BEGIN EXTRA
-                /*
-            case '£':
-                output[opos++] = 'G';
-                output[opos++] = 'B';
-                output[opos++] = 'P';
-                break;
-
-            case '€':
-                output[opos++] = 'E';
-                output[opos++] = 'U';
-                output[opos++] = 'R';
-                break;
-
-            case '©':
-                output[opos++] = '(';
-                output[opos++] = 'C';
-                output[opos++] = ')';
-                break;
-                */
+                case '£':
+                    output[opos++] = 'G';
+                    output[opos++] = 'B';
+                    output[opos++] = 'P';
+                    break;
+
+                case '€':
+                    output[opos++] = 'E';
+                    output[opos++] = 'U';
+                    output[opos++] = 'R';
+                    break;
```

Comment:
Maybe some other currency symbols as well? `$` = USD , `¥` = JPY

Reply:
I've added JPY, USD and the two ruppees. I might still miss a couple of them but not quite sure which one are relevant to include.